source-academy/cadet-frontend
# Mobile Sourcecast Workspace

### Description



- Rewritten the Sourcecast page from class to functional component
- Added the mobile version of the Sourcecast workspace

### Type of change


### How to test



- Connect your local frontend to the staging backend and play the test recording uploaded there
    - Audio and Editor should work as per normal during playback
    - The draggable Repl in the mobile workspace should popup when a recorded 'run' evaluation is played
- Sourcereel should also work as per normal
- Alternatively, the test recording can be accessed here https://sa-mobile.vercel.app/sourcecast/3c65b9ae-1450-43b8-bf4c-c9f6b0434daf

### Checklist
